unit/libcprover-cpp/verification_resultt.cpp

```diff
@@ -8,6 +8,7 @@
 #include <libcprover-cpp/verification_result.h>
 
 #include <iostream>
+#include <memory>
```

Comment:
Why has this file being touched by this PR?
It shouldn't be changed.

Reply:
removed, sorry for cluttering